src/mavlink/mavlink_camera.rs

```diff
@@ -410,7 +410,156 @@ fn receive_message_loop(
                             warn!("Failed to send video_stream_information: {error:?}");
                         }
                     }
+                    mavlink::common::MavMessage::PARAM_EXT_REQUEST_READ(param_ext_req) => {
+                        if param_ext_req.target_system != header.system_id {
+                            debug!(
+                                "Ignoring PARAM_EXT_SET, wrong system id: {}",
```

Comment:
It would be nice to display own own system and component id, you can also have a single if with: "wrong system/component id: {}.{}, waiting for: {}.{}"